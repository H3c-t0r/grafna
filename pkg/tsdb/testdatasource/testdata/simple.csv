Field1,Field2,Field3
A,5,6
B,6,7