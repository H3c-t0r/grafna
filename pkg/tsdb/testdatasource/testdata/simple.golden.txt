ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] 
Name: simple
Dimensions: 3 Fields by 2 Rows
+-----------------+------------------+------------------+
| Name: Field1    | Name: Field2     | Name: Field3     |
| Labels:         | Labels:          | Labels:          |
| Type: []*string | Type: []*float64 | Type: []*float64 |
+-----------------+------------------+------------------+
| A               | 5                | 6                |
| B               | 6                | 7                |
+-----------------+------------------+------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////0AEAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAFQAAAACAAAAKAAAAAQAAAC4/v//CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAANj+//8IAAAAEAAAAAYAAABzaW1wbGUAAAQAAABuYW1lAAAAAAMAAADgAAAAZAAAAAQAAAA+////FAAAADwAAAA8AAAAAAADATwAAAABAAAABAAAACz///8IAAAAEAAAAAYAAABGaWVsZDMAAAQAAABuYW1lAAAAAAAAAACi////AAACAAYAAABGaWVsZDMAAJr///8UAAAAPAAAAEQAAAAAAAMBRAAAAAEAAAAEAAAAiP///wgAAAAQAAAABgAAAEZpZWxkMgAABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAgAGAAAARmllbGQyAAAAABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAARAAAAEgAAAAAAAUBRAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAABAAAAAGAAAARmllbGQxAAAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAAGAAAARmllbGQxAAAAAAAA//////gAAAAUAAAAAAAAAAwAFgAUABMADAAEAAwAAAA4AAAAAAAAABQAAAAAAAADAwAKABgADAAIAAQACgAAABQAAACIAAAAAgAAAAAAAAAAAAAABwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAEAAAAAAAAAAIAAAAAAAAABgAAAAAAAAAAAAAAAAAAAAYAAAAAAAAABAAAAAAAAAAKAAAAAAAAAAAAAAAAAAAACgAAAAAAAAAEAAAAAAAAAAAAAAAAwAAAAIAAAAAAAAAAAAAAAAAAAACAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAgAAAAAAAABBQgAAAAAAAAAAAAAAABRAAAAAAAAAGEAAAAAAAAAYQAAAAAAAABxAEAAAAAwAFAASAAwACAAEAAwAAAAQAAAALAAAADgAAAAAAAMAAQAAAOABAAAAAAAAAAEAAAAAAAA4AAAAAAAAAAAAAAAAAAAAAAAKAAwAAAAIAAQACgAAAAgAAABUAAAAAgAAACgAAAAEAAAAuP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAADY/v//CAAAABAAAAAGAAAAc2ltcGxlAAAEAAAAbmFtZQAAAAADAAAA4AAAAGQAAAAEAAAAPv///xQAAAA8AAAAPAAAAAAAAwE8AAAAAQAAAAQAAAAs////CAAAABAAAAAGAAAARmllbGQzAAAEAAAAbmFtZQAAAAAAAAAAov///wAAAgAGAAAARmllbGQzAACa////FAAAADwAAABEAAAAAAADAUQAAAABAAAABAAAAIj///8IAAAAEAAAAAYAAABGaWVsZDIAAAQAAABuYW1lAAAAAAAAAAAAAAYACAAGAAYAAAAAAAIABgAAAEZpZWxkMgAAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAEQAAABIAAAAAAAFAUQAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABgAAAEZpZWxkMQAABAAAAG5hbWUAAAAAAAAAAAQABAAEAAAABgAAAEZpZWxkMQAA+AEAAEFSUk9XMQ==
